flexmeasures/api/v3_0/tests/test_assets_api.py

```diff
@@ -118,6 +119,28 @@ def test_get_assets(
         assert turbine["account_id"] == setup_accounts["Supplier"].id
 
 
+@pytest.mark.parametrize("requesting_user", ["test_admin_user@seita.nl"], indirect=True)
+def test_fetch_asset_sensors(client, setup_api_test_data, requesting_user):
+    """
+    Retrieve all sensors associated with a specific asset.
+
+    This endpoint returns a paginated list of sensors under the given asset.
+    The response will include metadata such as the total number of records and
+    filtered records when pagination is applied.
+    """
+    asset_id = 5
```

Comment:
Ideally, the 5 is not hardcoded but comes from a fixture like [this](https://github.com/FlexMeasures/flexmeasures/blob/main/flexmeasures/conftest.py#L380) (which you already have available through importing [this fixture](https://github.com/FlexMeasures/flexmeasures/blob/main/flexmeasures/api/v3_0/tests/conftest.py#L18), but would have to explicity import in this test to be able to make use of the returned asset information.

Reply:
Thanks for the reference i think I implemented it better now.